server/activityItem/fetch.ts

```diff
@@ -120,8 +121,13 @@ const applyFiltersToWhereQuery = (whereQuery: any, filters: ActivityFilter[]) =>
 const fetchActivityItemModels = async (
 	options: Required<FetchActivityItemsOptions>,
 ): Promise<types.ActivityItem[]> => {
-	const { scope, limit, offset } = options;
+	const { scope, limit, offset, since } = options;
 	const whereQuery = {
+		...(since && {
+			timestamp: {
+				[Op.gte]: since!,
```

Comment:
The `!` is not necessary here...TS will know that `since` is truthy on the right side of `since &&`